### ISSUE TITLE
Improve watched list by allowing list of characters

### ISSUE THREAD

**Issue by Tschis** [NabDev/NabBot#171]
Maybe accept list of strings separated by comma.

**Comment by Galarzaa90** [comment]
@Tschis You mean the add and remove subcommands? I like the idea.

**Comment by Tschis** [comment]
Yes.

From:
`/watchlist addchar Tschas`
`/watchlist addchar Tsches`
`/watchlist addchar Tschis`

To:
`/watchlist addchar Tschas. Tsches, Tschis`

But then it gotta report which ones were added and which ones were invalid (if any).
Also, we might reconsider confirmation in those less impactful commands.

**Comment by Galarzaa90** [comment]
Yeah, I think something similar to what `adduser` does could be done, at the end it just tells you which ones were added:
![image](https://user-images.githubusercontent.com/12865379/54059412-50628d00-41b6-11e9-999f-b1be9e38bf9a.png)
